### PR TITLE
Detect ResultSets and Entities in view render

### PR DESCRIPTION
**Pull request by alecho**

### CODE REVIEW

File: src/View/View.php [hunk 1]
```diff
@@ -769,6 +770,25 @@ public function loadHelpers()
...
 
     /**
+     *
+     *
+     */
+    public function decorateEntities() {
+        foreach($this->viewVars as $name => $var) {
+            if (is_object($var)) {
+                if ($var instanceof \Cake\ORM\Entity) {
+debug('ENTITY');
```
Comment (ADmad) [inline src/View/View.php]: Ehm...

**Comment by ADmad** [comment]
What is this supposed to fix?

**Comment by dereuromark** [comment]
@alecho you should always ref the original issue/ticket to avoid confusion.

**Comment by ADmad** [comment]
New features need to target 3.1 branch.

**Comment by ADmad** [comment]
I guess this ref #6698

**Comment by alecho** [comment]
Sorry. I hate that Github targets upstream when opening a new PR

**Comment by dereuromark** [comment]
Why opening one in the first place? ;)

**Comment by alecho** [comment]
It was supposed to be on our fork of the project. I don't have write access to it.

**Pull request closed without merging**